docs/index.md

```diff
@@ -0,0 +1,89 @@
+completion without or with significantly minimal human intervention. Machine
+intelligence augments human expertise through efficient modeling and
+prediction of flow outcomes during layout generation.
+
+The 24-hour runtime target implies that problems must be strategically
+decomposed throughout the design process, with clustered and partitioned
+subproblems being solved and recomposed through intelligent distribution
+and management of computational resources. This ensures that the NHIL design
+optimization is performed within its available `[threads * hours]` "box" of
+resources. Decomposition that enables parallel and distributed search over
+cloud resources incurs a quality-of-results loss, but this is subsequently
+recovered through improved flow predictability and enhanced optimization.
+
+For a technical description of the OpenROAD flow, please refer to our DAC paper:
+[Toward an Open-Source Digital Flow: First Learnings from the OpenROAD Project](https://vlsicad.ucsd.edu/Publications/Conferences/371/c371.pdf).
+The paper is also available from [ACM Digital Library](https://dl.acm.org/doi/10.1145/3316781.3326334).
+Other publications and presentations are
+[linked here](https://theopenroadproject.org/publications/).
+
+## Code of conduct
+
+Please read our code of conduct [here](main/CODE_OF_CONDUCT.md).
+
+## Documentation
+
+The OpenROAD Project has two releases:
+
+### [Application](https://github.com/The-OpenROAD-Project/OpenROAD)
+
+The application is a standalone binary capable to perform chip design
+from the floorplan to detailed route with metal fill.
+
+See documentation for the [application here](main/README.md).
+
+### [Flow](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts)
+
+The flow is a set of integrated scripts that allow for RTL-to-GDSII flow
+using open-source tools.
+
+See documentation for the [flow here](user/GettingStarted.md).
+
+## How to contribute
+
+If you are willing to **contribute**, see the
+[Getting Involved](contrib/GettingInvolved.md) section.
+
+If you are a **developer** with EDA background, learn more about how you
+can use OpenROAD as the infrastructure for your tools in the
+[Developer Guide](contrib/DeveloperGuide.md) section.
+
+## How to get in touch
+
+We maintain the following channels for communication:
+
+-   Project homepage and news: <https://theopenroadproject.org>
+-   Twitter: <https://twitter.com/OpenROAD_EDA>
+-   Issues and bugs:
+    -   OpenROAD: <https://github.com/The-OpenROAD-Project/OpenROAD/issues>
+    -   OpenROAD Flow: <https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/issues>
+-   Discussions:
+    -   OpenROAD: <https://github.com/The-OpenROAD-Project/OpenROAD/discussions>
+    -   OpenROAD Flow: <https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/discussions>
+
+See also our [FAQs](user/FAQS.md) and
+[Capabilities/Limitations](user/ToolLimitations.md) for relevant
+background on the project.
+
```

Comment:
Remove Capabilities/Limitations- out of date completely. This will be part of tool readme- unless there are OR levels limitations we should address.

Also, FAQs should also point to a common link. IS FAQ@.md updated? I believe this should point to Github Q&As

Reply:
> Remove Capabilities/Limitations- out of date completely. This will be part of tool readme- unless there are OR levels limitations we should address.

Fixed on 0dd5df9

> Also, FAQs should also point to a common link. IS [FAQ@.md](mailto:FAQ@.md) updated? I believe this should point to Github Q&As, maybe discussions

The discussion page includes the Q&As.